### DISCUSSION TITLE
Recording at higher fps

### DISCUSSION THREAD

**Discussion by bharatik-IISc** [microsoft/AirSim#3731] [Support Q&A]
### What's your question?

How can one record the images at a higher fps? 

### Include context on what you are trying to achieve

I am trying to construct a video from the images captured on recording, and would want that the images are captured at higher frame rate.

#### Context details

OS - Windows10
Neighbourhood Windows Binary v1.4.0

### Include details of what you already did to find answers
I have tried the Recording (RecordInterval) and the ClockSpeed option in settings.json

**Comment by AndreasKaratzas** [comment]
Could you include your whole `settings.json` file?

**Comment by bharatik-IISc** [comment]
Following settings.json was used. Another configuration had CLockSpeed Set at 10.


{
  "SeeDocsAt": "https://github.com/Microsoft/AirSim/blob/master/docs/settings.md",
  "SettingsVersion": 1.2,
  "SimMode": "ComputerVision",
   "Recording": {
    "RecordOnMove": false,
    "RecordInterval": 0.001,
    "Folder": "F:\\Dataset\\AirSim\\Trial Images",
    "Enabled": true,
    "Cameras": [
      {
        "CameraName": "0",
        "ImageType": 0,
        "PixelsAsFloat": false,
        "VehicleName": "",
        "Compress": true
      }
    ]

  },
  "CameraDefaults": {
    "CaptureSettings": [
      {
        "ImageType": 0,
        "Width": 1920,
        "Height": 1080
      }
    ]
  }

}

**Comment by AndreasKaratzas** [comment]
Try changing `RecordOnMove` setting to true. It is clarified in [settings.json tutorial](https://github.com/Microsoft/AirSim/blob/master/docs/settings.md), if `RecordOnMove` is false then the simulator doesn't capture the asset's feed while it's on move. I applied the change directly to your `json` file:
`
{
    "SeeDocsAt": "https://github.com/Microsoft/AirSim/blob/master/docs/settings.md",
    "SettingsVersion": 1.2,
    "SimMode": "ComputerVision",
    "Recording": {
        "RecordOnMove": true,
        "RecordInterval": 0.001,
        "Folder": "F:\Dataset\AirSim\Trial Images",
        "Enabled": true,
        "Cameras": [
            {
                "CameraName": "0",
                "ImageType": 0,
                "PixelsAsFloat": false,
                "VehicleName": "",
                "Compress": true
            }
        ]
    },
    "CameraDefaults": {
        "CaptureSettings": [
            {
                "ImageType": 0,
                "Width": 1920,
                "Height": 1080
            }
        ]
    }
}
`

I have to warn you though, that you are practically asking the simulator to render the environment 1000 times per second. You realize that you gotta have a lot of GPU power to do that. And gets worse when you increase clockspeed too.
